content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -7,118 +7,209 @@ title: Bring your own bucket (BYOB)
 
-<details>
-<summary>Configure an S3-compatible storage for team-level BYOB in Dedicated cloud or Self-managed instance</summary>
+For details, see [Create a CoreWeave AI Object Storage bucket](https://docs.coreweave.com/docs/products/storage/object-storage/how-to/create-bucket) in the CoreWeave documentation.
 
-Specify the path using the following format:
-```text
-s3://<accessKey>:<secretAccessKey>@<url_endpoint>/<bucketName>?region=<region>&tls=true
-```
-The `region` parameter is mandatory, except for when your W&B instance is in AWS and the `AWS_REGION` configured on the W&B instance nodes matches the region configured for the S3-compatible storage.
+1. Create the bucket with a name of your choice in your preferred CoreWeave availability zone. Optionally create a folder for W&B to use as a sub-path for all W&B files. Make a note of the bucket name, availability zone, API access key, secret key, and sub-path.
+1. Set the following Cross-origin resource sharing (CORS) policy for the bucket:
+    ```json
+    [
+      {
+        "AllowedHeaders": [
+          "*"
+        ],
+        "AllowedMethods": [
+          "GET",
+          "HEAD",
+          "PUT"
+        ],
+        "AllowedOrigins": [
+          "*"
+        ],
+        "ExposeHeaders": [
+          "ETag"
+        ],
+        "MaxAgeSeconds": 3000
+      }
+    ]
+    ```
+    CoreWeave storage is S3-compatible. For details about CORS, refer to [Configuring cross-origin resource sharing (CORS)](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enabling-cors-examples.html) in the AWS documentation.
+1. Configure a bucket policy that grants the required permissions for your W&B deployment to access the bucket and generate [pre-signed URLs]({{< relref "./presigned-urls.md" >}}) that AI workloads in your cloud infrastructure or user browsers utilize to access the bucket. Refer to [Bucket Policy Reference](https://docs.coreweave.com/docs/products/storage/object-storage/reference/bucket-policy) in the CoreWeave documentation. <!--Select your deployment type to view an example bucket policy.-->
 
-</details>
-<details>
-<summary>Configure a cross-cloud native storage for team-level BYOB in Dedicated cloud or Self-managed instance</summary>
+   <!--> <details>
+      <a id="coreweave-dedicated-cloud-self-managed-bucket-policy"></a><summary><b>Dedicated Cloud</b> / <b>Self-Managed</b> bucket policy</summary>-->
 
-Specify the path in a format specific to the locations of your W&B instance and storage bucket:
+    Replace:
+      - In `Resource`, each instance of `<cw_bucket>` with the CoreWeave bucket name.
+      - In `Principal`, each instance of `<cw_principal_and_role_arn>` with the CoreWeave principal and role ARN in the format `"arn:aws:iam:::user/<CW_ORG>:<CW_USER>"`. To apply the policy to all principals, replace the entire ARN with `*`. To apply the policy to all principals in your CoreWeave organization, replace `<CW_USER>` with `*`.
```

Comment:
```suggestion
```
I'm confused what this is for? Do you mind showing me? I don't think this is relevant for CoreWeave server unless I'm missing something here